BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  UIViewElement.swift
+//  BlueprintUI
+//
+//  Created by Kyle Van Essen on 6/4/20.
+//
+
+import UIKit
+
+
+///
+/// An element type  which makes it easier to wrap an existing `UIView` instance that
+/// provides its own sizing via `sizeThatFits`. An instance of the view is used for
```

Comment:
We should make it clear that we are caching the instance indefinitely, not creating one every time.

Reply:
will update